src/contracts/BMath.sol

```diff
@@ -208,17 +238,22 @@ contract BMath is BBronze, BConst, BNum {
   }
 
   /**
-   *
-   *    calcPoolInGivenSingleOut
-   *    pAi = poolAmountIn               // /               tAo             \\     / wO \     \
-   *    bO = tokenBalanceOut            // | bO - -------------------------- |\   | ---- |     \
-   *    tAo = tokenAmountOut      pS - ||   \     1 - ((1 - (tO / tW)) * sF)/  | ^ \ tW /  * pS |
-   *    ps = poolSupply                 \\ -----------------------------------/                /
-   *    wO = tokenWeightOut  pAi =       \\               bO                 /                /
-   *    tW = totalWeight           -------------------------------------------------------------
-   *    sF = swapFee                                        ( 1 - eF )
-   *    eF = exitFee
-   *
+   * @notice Calculate the amount of pool tokens that should be burned given a single token out when exiting a pool
```

Comment:
changing the phrasing here "should be burnT", is ok, but incoherent with the previous ones, no?